src/test/unit/lib/model/OutboundTransfersModel.test.js

```diff
@@ -733,40 +751,42 @@ describe('outboundModel', () => {
 
         await setConfig(defaultEnv);
         const conf = getConfig();
-        const cache = new Cache();
+        const cache = new Cache(dummyCacheConfig);
+        await cache.connect();
+
+        const expectError = {
+            type: 'quoteResponseError',
+            data: {
+                errorInformation: {
+                    errorCode: '3205',
+                    errorDescription: 'Quote ID not found'
+                }
+            }
+        };
+
 
         MojaloopRequests.__getParties = jest.fn(() => {
             // simulate a callback with the resolved party
-            cache.emitMessage(JSON.stringify(payeeParty));
+            cache.subscriptionClient.emitMessage('message', `${payeeParty.party.partyIdInfo.partyIdType}_${payeeParty.party.partyIdInfo.partyIdentifier}`, JSON.stringify(payeeParty));
```

Comment:
`${payeeParty.party.partyIdInfo.partyIdType}_${payeeParty.party.partyIdInfo.partyIdentifier}` used multiple times so it can be extracted into separate method:
```
const createPartySubKey = (payeeParty) => ${payeeParty.party.partyIdInfo.partyIdType}_${payeeParty.party.partyIdInfo.partyIdentifier}
```
and then
```
cache.subscriptionClient.emitMessage('message', createPartySubKey(payeeParty), JSON.stringify(payeeParty));
```
or even better
```
const emitPartyMessage = (cache, payeeParty) => cache.subscriptionClient.emitMessage('message', `${payeeParty.party.partyIdInfo.partyIdType}_${payeeParty.party.partyIdInfo.partyIdentifier}`, JSON.stringify(payeeParty));
```

Reply:
done